webapp/src/main/webapp/vue-apps/Admin/components/AdminApp.vue

```diff
@@ -1,86 +1,50 @@
-                        ? $t(`webconferencing.admin.${providerConfig.title}.name`)
-                        : providerConfig.title
-                      }}
-                    </div>
-                  </td>
-                  <td>
-                    <div 
-                      v-html="i18n.te(`webconferencing.admin.${providerConfig.title}.description`)
-                        ? $t(`webconferencing.admin.${providerConfig.title}.description`)
-                        : '' ">
-                    </div>
-                  </td>
-                  <td class="center actionContainer">
-                    <div>
-                      <v-switch
-                        :dense="true"
-                        :input-value="providerConfig.active"
-                        :ripple="false"
-                        v-model="providerConfig.active"
-                        hide-details
-                        color="#568dc9"
-                        class="providersSwitcher"
-                        @change="changeActive(providerConfig)" />
-                    </div>
-                  </td>
-                  <td class="center actionContainer">
-                    <i
-                      v-if="providerConfig.hasSettings"
-                      class="uiIconSetting uiIconLightGray"
-                      @click="providerConfig.provider.showSettings()"></i>
-                  </td>
-                </tr>
-              </tbody>
-            </template>
-          </v-simple-table>
-        </v-col>
-      </v-row>
-    </v-container>
+      <v-list-item class="px-0 mb-2 mt-4">
+        <v-list-item-content class="py-0">
+          <v-list-item-title class="my-0">
+            <h4 class="font-weight-bold mt-0">
+              {{ $t("webconferencing.admin.title") }}
+            </h4>
+          </v-list-item-title>
+          <v-list-item-subtitle class="pt-4">
+            <h4 class="my-0 text-color">{{ $t("webconferencing.admin.section.title") }}</h4>
+          </v-list-item-subtitle>
+        </v-list-item-content>
+      </v-list-item>
+      <v-list-item
+        class="pb-2 px-0"
+        dense
+        v-for="providerConfig in providersConfig"
+        :key="providerConfig.title">
+        <v-list-item-action class="me-4 mt-0">
+          <v-switch
+            :input-value="providerConfig.active"
+            v-model="providerConfig.active"
+            color="primary"
+            class="providersSwitcher"
+            @change="changeActive(providerConfig)" />
+        </v-list-item-action>
+        <v-list-item-content class="py-0">
+          <v-list-item-title class="subtitle-1 pt-2">
+            {{ $t(`webconferencing.admin.${providerConfig.title}.name`)
+              ? $t(`webconferencing.admin.${providerConfig.title}.name`)
+              : providerConfig.title }} 
+          </v-list-item-title>
+          <v-list-item-subtitle
+            v-html="$t(`webconferencing.admin.${providerConfig.title}.description`)
```

Comment:
why using v-html here ?

Reply:
> why using v-html here ?

For the inclusion of an HTML anchor element with a link found in the WebRTC connector description. However, as we're going to remove the webrtc connector, we can dispense with the use of v-html.

Reply:
Yes, please remove the v-html